src/Resources/doc/clients.md

```diff
@@ -49,6 +49,20 @@ csa_guzzle:
             # ...
 ```
 
+You can also ```extend``` another client.
+
+```yml
+csa_guzzle:
+    clients:
+        parent_one:
+            config: 
+                 headers: 
+                     Accept: application/json
+        my_client:
+            extends: parent_one
+            # ...
+```
+
```

Comment:
Todo list:
- Remove ```DefinitionDecorator```
- Add tag with attribute ```extends``` in ```Csa\Bundle\GuzzleBundle\DependencyInjection\CsaGuzzleExtension::processClientsConfiguration``` to handle both use cases.
- Update ```InheritancePass```  to take into account the attribute ```extends```